### PR TITLE
fix: use `where` delete pipeline reports

### PR DESCRIPTION
**Pull request by chengjoey**
#### What this PR does / why we need it:
fix delete pipeline reports

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda-org.erda.cloud/erda/dop/projects/387/issues/all?id=282260&issueFilter__urlQuery=eyJ0aXRsZSI6InJlcG9ydOihqOaVsOaNruWkquWkmumcgOimgea4heeQhiIsInN0YXRlcyI6WzQ0MDIsNzEwNCw3MTA1LDQ0MDMsNDQwNCw3MTA2LDQ0MDYsNDQwNyw0NDEyLDQ1MzgsNDQxMyw0NDE0LDQ0MTUsNDQxNl19&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&iterationID=1090&type=TASK)

#### Specified Reviewers:

/assign @your-reviewer

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English | Fix the bug that delete pipeline reports             |
| 🇨🇳 中文    |  修复了pipeline reports太多没有清除            |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/pipeline/dbclient/op_pipeline_report.go [hunk 1]
```diff
@@ -120,7 +120,7 @@ func (client *Client) DeletePipelineReportsByPipelineID(pipelineID uint64, ops .
 	session := client.NewSession(ops...)
 	defer session.Close()
 
-	_, err := session.ID(pipelineID).Delete(&spec.PipelineReport{})
```
Comment (sfwn) [inline modules/pipeline/dbclient/op_pipeline_report.go:123]: why `.ID` didn't work?

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/pipeline/dbclient/op_pipeline_report.go (hunk 1 above)
Comment (chengjoey) [reply modules/pipeline/dbclient/op_pipeline_report.go:123]: if use `ID` interface, will generate sql: `DELETE FROM `dice_pipeline_reports` WHERE `id`=? []interface {}{0x9896f2}`, pipelineid is not the primary key of pipeline_reports table

and pipelineID is not `string` type

**Review by sfwn — APPROVED** [review]

**Pull request merged**